include/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+
+add_library(mapbox-base INTERFACE)
+add_library(Mapbox::Base ALIAS mapbox-base)
+
+target_include_directories(mapbox-base
+    INTERFACE "${MAPBOX_BASE_EXTRAS}/expected-lite/include"
```

Comment:
Is this still needed?

Reply:
https://github.com/mapbox/mapbox-base/blob/kdn-dev/include/mapbox/io/io.hpp

Reply:
Can you please add `SYSTEM` here as well? This prevents the compiler/static analysis tools from complaining about the headers included from this path in projects that only uses `mapbox-base` as an external dependency.